src/arrayMethodJoin.js

```diff
@@ -4,9 +4,21 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let str = '';
```

Comment:
```suggestion
    let concatenatedString = '';
```
`str` is a really bad name for the variable. Let's use smth more meaningful